prov/tcp/src/tcpx_cq.c

```diff
@@ -37,13 +37,21 @@
 
 #define TCPX_DEF_CQ_SIZE (1024)
 
+static void tcpx_buf_pools_destroy(struct tcpx_buf_pool *buf_pools)
+{
+	int i;
+
+	for ( i = TCPX_OP_MSG_SEND ; i < TCPX_OP_CODE_MAX ; i++)
```

Comment:
nit: spacing is off some:
```
for (i = TCPX_OP_MSG_SEND; i < TCPX_OP_CODE_MAX; i++)
```
though I would start with i = 0, which handles the values being re-ordered.